book/src/framework/interactive.md

```diff
@@ -1,38 +0,0 @@
-# Interactive
-
-
-For non-technical users or those building with Chainlink products outside of Golang, we offer an interactive method to deploy a NodeSet.
```

Comment:
"For non-technical users or those building with" => if you don't have full context what's available you're building with